ingest/src/main/java/com/microsoft/azure/kusto/ingest/IngestClientImpl.java

```diff
@@ -191,7 +191,7 @@ public IngestionResult ingestFromStream(StreamSourceInfo streamSourceInfo, Inges
             );
```

Comment:
If compressionType != null we shouldn't compress the stream, also what about streams of parquet and orc?